Trading/Mode/dca_trading_mode/config/DCATradingMode_schema.json

```diff
@@ -0,0 +1,18 @@
+{
+  "type": "object",
+  "title": "DCATradingMode",
+  "properties": {
+    "amount_to_buy_in_reference_market": {
+        "title": "The amount of dollars (or unit of reference market) to buy on each transaction.",
+        "type": "number",
+        "multipleOf": 1.0,
```

Comment:
think we should remove this multipleof condition to allow buy very low amounts (in btc for example)

Reply:
Right :+1: